File: app/src/main/java/com/novel/read/event/GetCategoryTypeEvent.java
package com.novel.read.event;

import com.mango.mangolib.event.BaseEvent;
import com.novel.read.model.protocol.CategoryTypeResp;

/**
 * create by 赵利君 on 2019/6/18
 * describe:
 */
public class GetCategoryTypeEvent extends BaseEvent<CategoryTypeResp> {

    public GetCategoryTypeEvent(CategoryTypeResp result) {
        super(result);
    }

    public GetCategoryTypeEvent() {
    }
}

File: app/src/main/java/com/novel/read/event/GetCategoryTypeEvent.kt
package com.novel.read.event

import com.mango.mangolib.event.BaseEvent
import com.novel.read.model.protocol.CategoryTypeResp

/**
 * create by zlj on 2019/6/18
 * describe:
 */
class GetCategoryTypeEvent : BaseEvent<CategoryTypeResp> {

    constructor(result: CategoryTypeResp) : super(result) {}

    constructor() {}
}
